service/collector/collector.go

```diff
@@ -1,13 +1,11 @@
 package collector
 
 const (
-	gaugeValue float64 = 1
-	Namespace          = "chart_operator"
+	namespace = "chart_operator"
```

Comment:
`namespace` only belongs to collector itself, so changed it to small-case naming.